example/trace/otlpgrpc/example.go

```diff
@@ -31,6 +32,8 @@ import (
 	"google.golang.org/grpc/credentials/oauth"
 )
 
+var keepRunning = flag.Bool("keepRunning", false, "Set to true for generating spans at a fixed rate indefinitely. Default is false.")
```

Comment:
I believe the `--help` text will already print the default so it's not necessary to add it, but please double check if I'm wrong.

Reply:
Looks like `./example --help` does not print the default value.

```
Usage of ./example:
  -keepRunning
    	Set to true for generating spans at a fixed rate indefinitely.
```

Reply:
Seems like it only prints the default if the default is `true`